src/rgw/store/dbstore/sqlite/sqliteDB.cc

```diff
@@ -21,9 +21,34 @@
     ret = 0;					\
   } while(0);
 
+#define MAX_RETRIES 200 // Make it configurable
+
+#define EXECUTE_STMT(dpp, rc, stmt) \
+  do {                          \
+    int retries = 0;                  \
+      rc = stmt;                  \
+      retries++;                    \
+      if (retries > 1) {  \
+        sleep(0.5);                     \
```

Comment:
I'm not sure you can safely use sleep in a posix threaded environment, due to signal safety issues;  I'm sure you can use nanosleep(...), but I'd recommend (because c++) std::thread::sleep_for(...);